src/shouldInclude.js

```diff
@@ -1,8 +1,25 @@
 import _isFunction from 'lodash/isFunction'
+import _isArray from 'lodash/isArray'
 
 export const shouldInclude = (displayName, {include, exclude}) => {
-  let isIncluded = include.test(displayName)
-  let isExcluded = exclude.test(displayName)
+
+  let isIncluded;
+  let isExcluded;
+
+  if (_isArray(exclude)) {
+    isExcluded = false;
+    for (let i = 0; i < exclude.length; i++) {
+      const item = exclude[i];
+      if (item.test(displayName)) {
+        isExcluded = true;
+        break;
+      }
+    }
+  } else {
+    isExcluded = exclude.test(displayName)
+  }
+
+  isIncluded = include.test(displayName)
```

Comment:
The point of normalizing the options is to remove the need for conditional statements in the rest of the code. 👪